uwrt_mars_rover_utils/uwrt_mars_rover_xbox_controller/include/uwrt_mars_rover_xbox_controller/uwrt_mars_rover_xbox_controller_interface.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef XBOX_CONTROLLER_H
+#define XBOX_CONTROLLER_H
+
+#include <uwrt_mars_rover_xbox_controller/visibility.h>
+
+#include <rclcpp/rclcpp.hpp>
+#include <sensor_msgs/msg/joy.hpp>
+#include <uwrt_mars_rover_xbox_controller/msg/xbox_controller.hpp>
+
+namespace uwrt_xbox
+{
+using joy_msg = sensor_msgs::msg::Joy;
+using xbox_msg = uwrt_mars_rover_xbox_controller::msg::XboxController;
+class UWRTXboxController : public rclcpp::Node
+{
+public:
+  UWRT_MARS_ROVER_XBOX_CONTROLLER_PUBLIC
+  explicit UWRTXboxController(const rclcpp::NodeOptions & options);
+
+private:
+  rclcpp::Subscription<joy_msg>::SharedPtr joy_node_sub;
```

Comment:
Leaving a comment for myself here, should rename all private variables to have an underscore at the end of them